chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,782 @@
+	\url{https://dimewiki.worldbank.org/Matching}}
+methods use observable characteristics to construct
+sets of treatment and control units
+where the observations in each set 
+are as similar as possible. \index{matching}
+These sets can either consist of exactly one treatment and one control observation (one-to-one),
+a set of observations where
+both groups have more than one observation represented (many-to-many),
+or where only one group has more than one observation included (one-to-many).
+By now you can probably guess that 
+the result of the matching needs to be saved in the master dataset.
+This is best done by assigning an ID to each matching set, 
+and create a variable in the master dataset 
+with the ID for the set each unit belongs to.
+The matching can even be done before the randomized assignment,
+so that treatment can be randomized within each matching set.
+This is a type of experimental design.
+Furthermore, if no control observations were identified before the treatment,
+then matching can be used to ex-post identify a control group.
+Many matching algorithms can only match on a single variable,
+so you first have to turn many variables into a single variable
+by using an index or a propensity score.\sidenote{
+	\url{https://dimewiki.worldbank.org/Propensity\_Score\_Matching}}
+The \texttt{iematch}\sidenote{
+	\url{https://dimewiki.worldbank.org/iematch}}
+command in the \texttt{ietoolkit} package developed by DIME Analytics
+produces matchings based on a single continuous matching variable.
+
+%-----------------------------------------------------------------------------------------------
+\subsection{Structuring datasets over multiple time periods}
+
+Your data map and master dataset requirements also depends on
+whether you are using data from one time period or several.
+A study that observes data in only one time period is called
+a \textbf{cross-sectional study}.
+\index{cross-sectional data}
+This type of data is relatively easy to collect and handle because
+you do not need to track individuals across time.
+Instead, the challenge in a cross-sectional study is to
+show that the control group is indeed a valid counterfactual to the treatment group.
+
+A study that observes data in multiple time periods is called \textbf{longitudinal}\index{longitudinal data}
+and can either be a \textbf{repeated cross-sections}
+or a \textbf{panel data} study.\index{panel data} \index{repeated cross-sectional data}
+In repeated cross-sections,
+each successive round of data collection uses a new random sample
+of observations from the treatment and control groups,
+but in a panel data study the same observations are tracked and included each round.
+If you are using panel data study,
+then your data map must document how the different rounds will be merged or appended,
+and your master dataset will be the authoritative source of which ID that will be used. 
+
+You must keep track of the \textit{attrition rate} in longitudinal data,
```

Comment:
only with panel data. attrition over time doesn't make sense for repeated cross section (since observations naturally go in and out with cross section)

Reply:
good point, agree! simply changed to "panel data" in: 3cdba9ceda588cc227db87573e52f87c1c746992